clearblade.md

```diff
@@ -1343,8 +1345,13 @@ We define how the param `arguments` looks like for each database below.
 
 ### MongoDB 
 
-- arguments
-  - dbCommand: one of the below supported collection methods
```

Comment:
I don't think he asked you to get rid of these arguments thing, He just asked to rephrase it.